packages/web/src/components/gcds-signature/stories/properties.mdx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import * as Signature from './gcds-signature.stories';
+
+<Meta of={Signature} name="Events & properties" />
+
+# Events & properties
+
+<Canvas
+  of={Signature.Default}
```

Comment:
Can you create a separate story for the props table? Otherwise if you make any changes on the events & properties tab, it will also change the display of the default story on the overview page.